src/app/booking/Benefits.js

```diff
@@ -0,0 +1,95 @@
+
+const BENEFITS = [
+  {
+    image: "/invite-favourite-clubs-ooscca.png",
+    header: "Invite your favouriteclubs and providersto join OOSCCA",
+    subheader: "Spread the word",
+    description: "Do your children love their current clubs and activity providers? Help us make OOSCCA even better by inviting them to join our platform. It's a simple way to keep all your bookings in one place and to support your favourite providers with the tools they need to thrive. Just enter their name and email, and we'll introduce them to the world of OOSCCA.",
+    linkText: "Let’s get more providers. Let’s get more choice",
+    link: "#",
+  },
+  {
+    image: "/new-feature-on-the-horizon.png",
+    header: "New Features on the Horizon",
+    subheader: "Exciting Futures Ahead",
+    description: "At OOSCCA, we are continuously evolving, crafting an experience that grows with your family. Our platform thrives on innovation, and we're excited to announce that new features and updates are just around the corner—designed to make your OOSCCA experience even more seamless and enjoyable. We're more than just a platform; we're a community. And every community thrives on collaboration. Share your suggestions with us, and let's build the future of OOSCCA together.",
+    linkText: "Suggest a feature",
+    link: "#",
+  },
+  {
+    image: "/spread-the-word-invite.png",
+    header: "Spread the word:Invite your favourite clubs and providers",
+    subheader: "Simplify your search",
+    description: "Do your children love their current clubs and activity providers? Help us make OOSCCA even better by inviting them to join our platform. It's a simple way to keep all your bookings in one place and to support your favourite providers with the tools they need to thrive. Just enter their name and email, and we'll introduce them to the world of OOSCCA.",
+    linkText: "Fill in the detail",
+    link: "#",
+  },
+]
+
+function BenefitText({benefit, sx}) {
+  return (
+    <Box {...sx}>
+      <Typography variant="subheading" color="orange.main">
+        {benefit.subheader}
+      </Typography>
+      <Typography mt={2} variant="h3">
+        {benefit.header}
+      </Typography>
+      <Typography my={2} variant="body1" color="text.secondary">
+        {benefit.description}
+      </Typography>
+      <Link
+        href={benefit.link}
+        underline="none"
+        color="purple.main"
+        variant="benefitLink"
+        sx={{
+          fontSize: "1rem", // 16px
+          fontWeight: 700,
+      }}>
+        {benefit.linkText.trim()} {"  >"}
+      </Link>
+    </Box>
+  )
+}
+
+export default function Benefits() {
+  return (
+    <Box backgroundColor={{ xs:"grey.100", md: "white" }}>
+      <Container py={3}>
+        <Box sx={{
+          display: "flex",
+          flexDirection: "column",
+          gap: 4,
+          maxWidth: {xs: 500, md: "100%"},
+          mx: "auto",
+        }}>
+          {BENEFITS.map((benefit, index) => (
```

Comment:
First image not aligned
<img width="1141" alt="image" src="https://github.com/digital-brother/ooscca-fe/assets/43747573/bf96407d-d4e2-4ac4-a9b5-df9feb3e9b90">

Reply:
Last image looks ugly at least.
<img width="612" alt="image" src="https://github.com/digital-brother/ooscca-fe/assets/43747573/d3df10b7-e7b8-4ef9-9c9d-e893d576a0e9">

We discussed, that we do not need form here, only image.